src/lib.rs

```diff
@@ -798,6 +871,18 @@ impl Parse for BitsAttr {
                     attr.into = Some(input.parse()?);
                 } else if ident == "from" {
                     attr.from = Some(input.parse()?);
+                } else if ident == "access" {
+                    let mode = input.parse::<Ident>()?;
+
+                    if mode == "RW" {
```

Comment:
This function becomes quite complex.
What if we implement Parse for AccessMode and put this parsing code there?
Also, we might want to have a meaningful error if the attribute has an invalid value (not just silently ignore it).

Reply:
Thought about using an extra Parse implementation aswell...the fact I missed giving an error message here convinced me enough to do it.